app/controllers/ops_controller/settings/common.rb

```diff
@@ -182,6 +182,7 @@ def pglogical_save_subscriptions
       task_opts = {:action => "Save subscriptions for global region", :userid => session[:userid]}
       queue_opts = {:class_name => "MiqPglogical", :method_name => "save_global_region",
                     :args       => [subscriptions_to_save, subsciptions_to_remove]}
+      ActiveRecord::Base.yaml_column_permitted_classes = ActiveRecord::Base.yaml_column_permitted_classes | [subscriptions_to_save.first.class, subsciptions_to_remove.first.class]
```

Comment:
Minor, but you can one-shot this

```suggestion
      ActiveRecord::Base.yaml_column_permitted_classes |= [subscriptions_to_save.first.class, subsciptions_to_remove.first.class]
```

Reply:
it looks like an emoji... -1 NACK.  🤣

Reply:
Honestly, these are really out there to show what would be required to do `config.active_record.use_yaml_unsafe_load = false` in the application.  I feel like the proper solution is to not serialize these objects so anything easy to grep can enable that in the future.

I can change it.  Just giving background as there are a few like this.

Reply:
ok, fixed